blueprints/packages/files/packages/.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think we should have both a `.jshintrc` _and_ an `.eslintrc`, can you remove the `.jshintrc.js` file?

Reply:
👍 

I should remove this from the `lib` blueprint too. Happy for me to do this in a follow on PR so that this remains focused?